src/cookbook/register-roles.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Register Roles | Cookbook"
+head:
+  - - meta
+    - name: description
+      content: "Learn how to register roles in Iroha."
+  - - meta
+    - name: keywords
+      content: "registering roles, permissions"
+---
+
+# How to Register a Role
+
+The minimal case is an empty role (without any permission tokens):
+
+```rust
+fn register_new_role(
+    iroha: &Client
+) {
+    let role_id = RoleId::from_str("MY_EMPTY_ROLE").unwrap();
+    let role = iroha_data_model::role::Role::new(role_id);
```

Comment:
```suggestion
    let role = Role::new(role_id);
```

this is always implied in other examples. Anyways, it should be `iroha_client::data_model`, not `iroha_data_model`